Tools/WebKitTestRunner/ios/UIScriptControllerIOS.mm

```diff
@@ -882,6 +922,22 @@ static void clipSelectionViewRectToContentView(CGRect& rect, UIView *contentView
     return JSValueToObject(jsContext, [JSValue valueWithObject:toNSDictionary(frameInContentViewCoordinates) inContext:[JSContext contextWithJSGlobalContextRef:jsContext]].JSValueRef, nullptr);
 }
 
+JSObjectRef UIScriptControllerIOS::selectionEndGrabberViewShapePathDescription() const
+{
+    UIView *contentView = platformContentView();
+    UIView *handleView = nil;
+
+#if HAVE(UI_TEXT_SELECTION_DISPLAY_INTERACTION)
+    UNUSED_PARAM(contentView);
```

Comment:
Nit - I would remove this and just replace the `contentView` below with `platformContentView()`.